ApprovalTests.Swift/Namer/Namer.swift

```diff
@@ -13,9 +15,8 @@ public class Namer: ApprovalNamer {
     }
 
     public func sourceFilePath() -> String {
-        let names = StackDemangler().extractNames()
         let baseName = (fileName as NSString).deletingPathExtension
-        return baseName + "." + names.testName
+      return baseName + "." + "\(functionName.dropLast(2))"
```

Comment:
What does `.dropLast(2)` achieve? You could extract this a variable to describe the result.

Also for formatting:
```suggestion
        return baseName + "." + "\(functionName.dropLast(2))"
```

Reply:
Thank you for pointing this out. We extracted an explaining variable in 30eba989f6b3b1e9ceaedd5acf20cce131e2ff1e